web/impact/impact/tests/test_minimal_mail_handler.py

```diff
@@ -0,0 +1,43 @@
+from django.test import TestCase
+from django.core import mail
+from impact.minimal_email_handler import MinimalEmailHandler
+
+
+DEFAULT_PARAM_KEYS = ("to",
+                      "subject",
+                      "body")
+
+DEFAULT_PARAM_VALS = (["a@b.com"],
```

Comment:
I'd prefer if we used `@example.com` addresses rather than potentially deliverable ones.